Source/Classes/NantesLabel.swift

```diff
@@ -101,9 +101,12 @@ public extension NSAttributedString.Key {
     /// nil or [:] will add no styling
     open var activeLinkAttributes: [NSAttributedString.Key: Any]?
 
-    /// A token to use when the label is truncated in height. Defaults to "\u{2026}" which is "…"
+    /// A token to use when the label is truncated in height. Defaults to `backupTruncationToken`
     open var attributedTruncationToken: NSAttributedString?
 
+    /// The string used when there's no attributedTruncationToken set. Defaults to "…"
+    open var backupTruncationToken: String = "\u{2026}"
```

Comment:
why does this need to be open?

Reply:
So you can override it in a subclass if you want to default it to something else.

Reply:
Sorry, it was too early in the morning when I first saw this ☕️